docs/ExpectAPI.md

```diff
@@ -77,6 +77,34 @@ declare global {
 }
 ```
 
+Alternatively, you can use the type `ExtendedExpect` and there is no danger of incorrectly typing the matchers.  This will also make available all applicable custom matchers as asymmetric matcher creators.  Asymmetric matchers are covered later.
+
+```ts
+const extendedMatchers = {
+  toBeWithinRange(received:any, floor:number, ceiling:number) {
```

Comment:
can you run prettier on the markdown file? `yarn lint:prettier` from root should do it

Reply:
Yes will do when we are agreed upon the change and I apply to the versioned docs as well.